gulpfile.js

```diff
@@ -100,6 +101,8 @@ gulp.task('scripts', ['lint'], function scriptsTask() {
   return bundledStream;
 });
 
+gulp.task('test', shell.task('npm run test:browser:live'));
```

Comment:
It seems funny to use both gulp and npm for running the same set of tasks. Is there any reason we want it in both places, or can we standardize on one?